### PR TITLE
Feature/gh 1193

### PR DESCRIPTION
**Pull request by developpeurweb**
fixes GH-1193

### CODE REVIEW

File: projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  </div>
+
+  <div class="form-group">
+    <label>
+      <span class="label-content">Confirm Email address</span>
+      <input
+        type="email"
+        class="form-control"
+        formControlName="userId"
+        placeholder="Enter email"
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html]: According to the mocks, the placeholder should be "Confirm email"

File: projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    </label>
+  </div>
+
+  <div class="form-group">
+    <label>
+      <span class="label-content">Confirm Email address</span>
+      <input
+        type="email"
+        class="form-control"
+        formControlName="userId"
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html]: Could you change this formControlName so it's different to the first one please. So something like `userIdConf`

File: projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@import 'theme';
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.scss]: Do we need this import here?

Comment (developpeurweb) [reply projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.scss]: seems we don't, removing

File: projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html (hunk 1 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html]: done

File: projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html (hunk 2 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html]: done

**Pull request merged**